services/elastic/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '2.2'
+
+services:
+  elastic:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.13.1
+    volumes:
+      - esdata:/usr/share/elasticsearch/data
+    labels:
+      - traefik.http.routers.elastic.rule=Host(`elastic.localhost`)
+      - traefik.http.services.elastic.loadbalancer.server.port=9200
+    mem_limit: 1073741824
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    healthcheck:
```

Comment:
is this needed? Elastic does not come with a default health check?

Reply:
It doesn't I think, the recommended docker compose file also has this (with a slightly different formulation)

Reply:
I mean it can be simplified but the health check checks whether the response is also sensible. Ofc I can simplify it by writing `curl -sf localhost:9200 > /dev/null` instead, but the response is not checked in this case.

Reply:
yes, sorry I just meant to write it like this: 
```
curl -f localhost:9200 | grep -q
            ''"tagline" : "You Know, for Search"''
```
and remove the CMD + []